pkg/security/podsecuritypolicy/group/helpers.go

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package group
+
+import (
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/util/validation/field"
+	"k8s.io/kubernetes/pkg/apis/policy"
+	psputil "k8s.io/kubernetes/pkg/security/podsecuritypolicy/util"
+)
+
+func ValidateGroupsInRanges(fldPath *field.Path, ranges []policy.IDRange, groups []int64) field.ErrorList {
+	const errFmt = "group %d must be in the ranges: %v"
+	allErrs := field.ErrorList{}
+
+	for _, group := range groups {
+		for _, rng := range ranges {
```

Comment:
Is this correct? I thought the group would be required to fall within one of the ranges (like isGroupValid checks), not required to fall within all of them

Reply:
Hm, you're right, I just blindly copy-pasted the code from `MustRunAs` originally, but it would be my impression that falling in just one of the ranges should be enough. I'll definitely add a test for that. It's weird that noone noticed this up until now.

Reply:
Actually, never mind, it seems the original code was correct and I am missing a break here.